chainer/function.py

```diff
@@ -172,11 +172,16 @@ def __call__(self, *inputs):
 
         """
 
-        inputs = [x if isinstance(x, variable.Variable)
-                  else variable.Variable(x)
-                  for x in inputs]
+        no_variable = all([not isinstance(x, chainer.Variable)
+                           for x in inputs])
```

Comment:
I think `chainer.Variable` should be `variable.Variable`. (rewritten as such in the conflicted commit)